packages/studio-web/src/app/editor/editor.component.html

```diff
@@ -6,7 +6,7 @@
         <h1 i18n="Welcome message for app editor" id="welcome-header">
           Welcome to the ReadAlong Studio Editor
         </h1>
-        <p>
+        <p i18n="Editor synopsis">
```

Comment:
Only loosely related to the rest of this PR: I notice this text was still in English on the fr and es versions.